### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD007.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#11] [General]
# https://blog.3place.cn/3place/fsd/FSD007.html

https://blog.3place.cn/3place/fsd/FSD007.html

**Comment by picklecai** [comment]
梅菲斯特去而复返，就是为了说明他是不能被束缚住的，是自由的。如果限制了他的自由而想和他订立任何契约，那都不符合法律精神：强迫之下订立的契约无效。

**Comment by picklecai** [comment]
要只顾嬉游，我已太老，
要无所要求，我又太年轻。

好纠结的老年人。造成这一切的，大概是不可知论吧？或者说求知的碰壁感。

**Comment by picklecai** [comment]
你要克己！要克己！
这是一句永远的老调，
在人人的耳边喧嚷，
我们一生，随时都听到
这种声嘶力竭的歌唱。

看来无论东方西方，要想社会安稳，都得首倡“仁”——要管束住自然欲望这匹野兽，才能在人群中顺利。束缚久了，难免想挣脱开。于是有了和魔鬼的订约。

**Comment by picklecai** [comment]
想到今日，在这一天之中，
一个愿望也不会实现，一个也不行，
甚至任何快乐的向往
也被任意的挑剔打消，
活跃的满腔创新的思想
都受到无数俗虑的干扰。

毫无希望和向往，不会有任何新鲜的体验，新想法刚冒头就被自己或别人掐断。这就是“有的人活着，他已经死了”，“35岁死了、70岁才埋葬”。
好灰暗呀！这样的状态，对人世有何可眷恋？

**Comment by picklecai** [comment]
驻在我的胸中的神，
能深深激动我的内心，
但这支配我全部力量的神，
却没有对付外力的本领。

“我”遇到了大麻烦，它一如既往地努力工作，却没有办法解决主人的新烦恼。这就让外界力量有机会“趁虚而入”。

**Comment by picklecai** [comment]
你要说上三遍。

不是急切，而是魔鬼的规则。
这个魔鬼，是个特别讲究规则的魔鬼，契约精神和法律意识很强。从前面的从哪条路进来就得从哪条路出去，到后面的必须滴血契约才能真正生效这些，都可以佐证。

遵守规则的前提下，他取走任何东西都是应得的，不会犹豫，不会有负罪感……

**Comment by picklecai** [comment]
可是死亡决不是受人欢迎的客官。

连魔鬼都知道，死不能解决任何问题。人何其傻！

**Comment by picklecai** [comment]
头上戴着血淋淋的桂冠、
荣获胜利的死者，真是福人！
还有拚命狂跳了一番而死于少女怀中的人！
我也曾醉心于崇高的地灵之力，
那时真应当失魂丧命！

浮士德列举了三种死：
为了理想而重于泰山的死，
为了肉体享乐的“牡丹花下死做鬼也风流”，
为了精神享乐的顿悟沉迷。

此刻，他大概觉得，哪种都行，都好过目前绝望地活着。

**Comment by picklecai** [comment]
我先要诅咒傲慢的思想，
它紧紧束缚我们的精神，

什么思想会束缚精神？道、德、仁、义，这些“修炼”的目标。

**Comment by picklecai** [comment]
诅咒那种最高的宠爱！
诅咒希望！诅咒信仰。

倘若没有这些，就不会对这苦难的人世间有一丝留恋，也就能放心去了。对于原本打算饮下棕色的毒汁的浮士德来说，积极阳光，也是阻碍。能带来快感的，只有毁灭。

**Comment by picklecai** [comment]
他们劝你寻乐和力行，
说得多聪明！
他们要劝你
去遍历人世。

寻乐、力行、遍历人世。
享受能享受的，做自己能做的，见自己能见的。如此不再空虚无聊，也不再被失败感笼罩。

**Comment by picklecai** [comment]
来日方长，现在不必提起。

梅菲斯特这句话，多像行贿的人的口吻。

**Comment by picklecai** [comment]
我不考虑什么来世，
……
有一日我跟它们分离，
管它有什么变化发生。
我也不想多管闲事，
管它将来有没有爱憎，
管它那个未来的人世，
是否还有上下之分。

王熙凤说：我是不信什么因果报应的。

**Comment by picklecai** [comment]
作着崇高努力的人类精神生活，
几时曾被尔等认清？

浮士德的不满在于精神。他不相信梅菲斯特可以满足这种需求，只觉得对方能提供的仅仅一切物质享乐而已。所以，他像个想象皇后日子的农妇一样，猜测梅菲斯特能带来什么。

**Comment by picklecai** [comment]
你能用甘言哄骗住我，
使我感到怡然自得，
你能用享乐迷惑住我，
那就算是我的末日！

核心赌约出现了。会是什么能让不可知论的博士精神上得到满足呢？

**Comment by picklecai** [comment]
言语写成字，已失去生命。

今天印象最深刻的一句。
磅礴如雷的思想，在化为语言的那一刻，已经静止。
无论多少活跃，只要说出口或写下来，就成定局。

**Comment by picklecai** [comment]
你立刻说上这一大篇 干吗夸张得如此激烈 你用任何纸条都行 还要用一滴血签上你的大名。

梅菲斯特：形式不重要，但法律程序绝不可少。
无情的梅菲斯特，崇尚法律、规则，是个绝对理性的角色。但偏偏他的目标是要破坏理性、打破规律。用他自己的话说，这是啃了几千年也难以啃动的一块硬骨头。这是这个角色的矛盾点吧？

**Comment by picklecai** [comment]
思想的线索已经断头 知识久已使我作呕。

浮士德想要喝毒酒的原因，是突然发现花了一辈子心血的事情，都是无用的、不可理解的？生命白活了。

**Comment by picklecai** [comment]
不给你规定条条框框
你高兴，
可以到处抓取逃跑时，
也尽可以窃取你喜欢的，
就请你品尝
大胆地动手，不要畏缩！

这是自由探索之路。
家长还不如魔鬼更懂

**Comment by picklecai** [comment]
我的心胸，求知欲已告熄灭
今后对任何痛苦都视若等闲

浮士德打定了主意，抛弃过去所有积累，重新活一次，先从满足基本需求开始。

**Comment by picklecai** [comment]
我要在我内心里自我体验
用这种精神掌握高深的至理
把幸与不幸堆积在我的心里
将我的小我扩充为人类的大我
最后我也像人类一样没落。

有时就会有这种“我就是群体”的错觉。好像群体和我，同呼吸共命运了一样。

**Reply by picklecai** [reply]
浮士德要试的是什么？

**Comment by picklecai** [comment]
我已把这一种坚硬的食
咬了几千年，这是实话
从睡进摇篮以至入土
无人能将这团旧酵消化
请相信我们，这整块食
做出来专供神明享用

食，是什么？人？人的精神？

**Reply by picklecai** [reply]
浮士德要试的是什么？

**Comment by picklecai** [comment]
时间苦短促，学艺无止境

看来不止庄子说了“吾生也有涯，而知也无涯”呀。

**Comment by picklecai** [comment]
我把人类精神的一切财/徒然集于一身，我也知详 而到最后，/瞧我坐在此处/内心却涌现不出新的力量/我自己并未提高一分/跟无限无量也毫未相近。

我把人类精神的一切财富
徒然集于一身，我也知详，
而到最后，瞧我坐在此处，
内心却涌现不出新的力量；
我自己并未提高一分，
跟无限无量也毫未相近

这是求生欲枯竭的表现。

**Comment by picklecai** [comment]
我一停滞，就变成奴隶，你的，别人的，都是一样。

突然发现这句话无关乎订约。只要停滞，那就是把自己交出去了，交给谁，都是一样，反正自己不“动”了。

**Comment by picklecai** [comment]
一个人只顾沉思
就像个牲口，在荒野地
被一个恶灵牵着，来回乱兜圈子
不知外围有美丽青葱的牧场

这就是思而不学则殆。

**Comment by picklecai** [comment]
让你去蔑视理性、知识
人类拥有的最高的实力
让你沉迷于魔术幻术
获得诳骗精灵的鼓舞

这是说：当一个人持反智主义时，容易被魔鬼趁虚而入。或者扩大一点，只要对过去很长时间以来的言行信仰产生怀疑，都是被魔鬼趁虚而入的“机会”。但如果没有这种怀疑的发生，又不会有重大突破。所以，天山童姥每次返老还童时，都是最虚弱的时候。这个过程结束后，总能有大长进。

**Reply by picklecai** [reply]
理性和知识是人类拥有的最高的实力，这句话就是“好古，敏以求之”。
魔术、幻术、诓骗精灵……这些是怪力乱神，无法证实或证伪。
不能踏踏实实去探究，就容易陷入虚无、幻象，会去相信“神迹”、“超自然力量”。

**Comment by picklecai** [comment]
何必苦苦地空打无穗的麦秸？

我们很多时候所花的时间精力，无非就是指望从无穗子的麦秸上获得粮食。

**Comment by picklecai** [comment]
命运已经赋予他一种精力
永远向前直闯，不受拘束
这种过分轻率的努力跳越过尘世的欢情乐趣
我要拖住他过浪荡生活
经历平凡的无聊事件
让他挣扎、发呆、粘着

锁住心的唐僧算什么？打开牢笼，自由选择，反复沉沦，最后仍然坚决西行，这才是人情。所以《西游记》原著这个唐僧，不好看，至少在求道的路上，不好看。永远坚决西行，又永远怪罪于各位徒弟。既无挣扎，也无担当。
浮士德，是复。
唐僧，不是。

**Comment by picklecai** [comment]
我要拖住他过浪荡生活
经历平凡的无聊事件
让他挣扎、发呆、粘着
再对付他的贪得无厌
拿酒食对着他贪婪的口唇晃摇
他哀求疗愈饥渴，也是无益
这样，即使他没向恶魔卖身投靠
他也一定要归于毁灭！

梅菲斯特的思路，和《水浒传》王婆十条计策，很相似。从前看到那一段，就觉得，潘金莲太难！要想从这样的牢笼里脱出来，几乎不可能。

**Comment by picklecai** [comment]
最初先去听逻辑课程
精神训练
服服帖帖
就像套进西班牙长靴
以后走上思维之路
就会踏着慎重的脚步
不会弄得像鬼火一样
闪来闪去，东飘西荡

梅菲斯特的教程：逻辑学给哲学打基础。
这个观点，不是和昨天说的维特根斯坦很像？

**Comment by picklecai** [comment]
要想认识、记述活动的对象的人
先要驱逐其中的精神
他能把部分掌握在手里
可惜，只缺少精神的联系
在化学上称之为“自然操作”

连同下面的名词卖弄，梅菲斯特给学生的要点，就是如何“不懂装懂”。
这是梅菲斯特对现实的讽刺。
也许，真正的浮士德，面对学生，也不过如此教法。所以，他才会觉得一辈子过得如此可厌，毫无价值。不如一杯毒酒了事。

**Reply by picklecai** [reply]
你要把形而上学好好钻研
这样，钻不进头脑的东西
你就能掌握其深远的意义
不管钻得进还是钻不进
漂亮的名词总很要紧

维特根斯坦的批判

**Comment by picklecai** [comment]
仿佛永难治愈的疾病
拖拖拉拉地从上代传到下代
又一处一处缓缓移行
合理变悖理，好事变灾祸
后世的子孙也算晦气
我们生来就具有的权利
真遗憾，从来无人提过。

此时的梅菲斯特，卢梭附体了。

**Reply by picklecai** [reply]
当个圣人，和当个魔鬼，有个共同之处：他们都懂社会的弊病。成佛成魔，一念之间。因为除了这一念，其他的见识和能力，是差不多的。甚至因为魔鬼的破坏倾向，毫无顾忌，能力外显上还更厉害一些。

**Comment by picklecai** [comment]
不行，恶魔奉行利己主义，决不会轻易免费效劳，
去干有利于他人之事。
请你讲明你的条件；
这样的仆人给家中带来危险。

世人对魔鬼的判断是带有误解的。总以为魔鬼时时作恶、处处作恶。其实并非如此。你满怀戒心时对方不一定怎么样，在你没注意时却悄然深驻，难以拔除。

**Comment by picklecai** [comment]
我如有一天悠然躺在睡椅上面，
那时我就立刻完蛋！
你能用甘言哄骗住我，使我感到怡然自得，你能用享乐迷惑住我，那就算是我的末日！

如果我对某一瞬间说：停一停吧！
你真美丽！
那时就给我套上枷锁，
那时我也情愿毁灭！

浮士德为什么会以此为赌约呢？
